src/main/java/com/banking/bankservice/service/impl/AccountServiceImpl.java

```diff
@@ -33,10 +38,8 @@ public Account getByAccountNumber(String accountNumber) {
     }
 
     @Override
-    public void blockAccount(String accountNumber) {
-        Account accountInDb = getByAccountNumber(accountNumber);
-        accountInDb.setActive(false);
-        accountRepository.save(accountInDb);
+    public Account update(Account account) {
```

Comment:
to me it is disputable if we should have methods with different names but identical implementation

`save` operation is designed to both create and update

Reply:
I have decided to add it just for semantic purposes. I could name it `saveOrUpdate()`